### PR TITLE
fix(curriculum): expand description

### LINKED ISSUE #53219: Add explanation to directions in step 143 for why the following updates are necessary
### Describe the Issue

A [forum user](https://forum.freecodecamp.org/t/improvement-suggestion-for-rpg-step-143/664091)brought up a good point, that we don't really explain why we are making the following changes even the game is technically complete.

Right now we just say the following:

```md
Your game could be complete now, but you can also make it more interesting.
```
Then we spend the next few steps telling them to write some code.

But they don't know why they are writing the code and how these changes will enhance the game.

So we should add a sentence or two explaining the nature of these changes so they can better understand why we didn't just end the game right here.
Otherwise, it just feels like they are just writing code without understanding why and how it affects the game in the end.

Looking for suggestions on how to update the wording

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-basic-javascript-by-building-a-role-playing-game/step-143

### Your code

see explanation above

### Expected behavior

see explanation above

### Screenshots

_No response_

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by VedangSardessai**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #53219 

I have made changes as suggested by @jdwilkin4, it resolves the issue of not explaining the why changes are needed while the game is complete

### CODE REVIEW

**Review by gikf — APPROVED** [review]
LGTM :+1:

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62aa1cea594f152ba626b872.md [hunk 1]
```diff
@@ -7,7 +7,8 @@ dashedName: step-141
 
 # --description--
 
-Your game could be complete now, but you can also make it more interesting.
+While your game is feature complete at this stage, there are things you can do to make it more fun and engaging. To get started, you'll give `monsters` a dynamic attack value.
+
```
[the remark this answers is not in the document]
Comment (VedangSardessai) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62aa1cea594f152ba626b872.md:11]: Oh my bad... I will remove the extra empty line! Thanks for your feedback

**Review by huyenltnguyen — APPROVED** [review]
LGTM 👍

@VedangSardessai Thank you for the PR and congratulations on your first contribution to this repo 🎉
We look forward to your next.

Happy contributing!

**Comment by VedangSardessai** [comment]
Thanks @huyenltnguyen  looking forward to making more PRs...Cheers!

**Pull request merged**